alpha/lib/model/category.php

```diff
@@ -1755,4 +1780,50 @@ public function indexCategoryInheritedTree()
 	{
 		kEventsManager::raiseEventDeferred(new kObjectReadyForIndexInheritedTreeEvent($this));
 	}
+
+	/**
+	 * @param $fullIds
+	 * @return array
+	 */
+	private static function getCategoriesStartWithIds($fullIds)
+	{
+		if (!isset($fullIds) || empty($fullIds))
+			return array();
+
+		$filter = new categoryFilter();
+		$filter->setFullIdsStartsWith($fullIds);
+		$c = KalturaCriteria::create(categoryPeer::OM_CLASS);
+		$filter->attachToCriteria($c);
+		$c->applyFilters();
+		$categoryIds = $c->getFetchedIds();
+		return $categoryIds;
+	}
+
+	/**
+	 * @param $categoriesIds
+	 */
+	private static function updateDeletedCategoriesInDB($categoriesIds)
```

Comment:
This method is an incomplete functionality, either raise the events here as well, or move this functionality to deleteChildCategories, anyway, it's a bad name for a method, if you can't give it a functional name then it's probably doesn't deserve a method.